src/pages/MissionAndPurpose/MissionAndPurpose.scss

```diff
@@ -1,14 +1,20 @@
-.about-mission-purpose{
-  &__body{
-    margin-bottom: 4.5rem;
-
+.about-mission-purpose {
+  .body {
     h4 {
       font-weight: 700;
+      text-indent: 20px;
       text-transform: uppercase;
     }
```

Comment:
h4 styling should be global

Reply:
The h4 is inconsistent from the other h4s. The other way I could do it would be to go into the wordpress and write the HTML as a seperate class and then style that.

Then I wouldn't need the .about-mission-purpose. What do you think